### DISCUSSION TITLE
Trailing comma lint triggered on 'compact' multiline arrays

### DISCUSSION THREAD

**Discussion by delta1186** [tightenco/tlint#258] [Q&A]
Hey @loganhenson currently when you run tlint on a laravel project the snip below will throw the following lint issue: ! Multiline arrays should have trailing commas

```
'projects' => Project::active()
                        ->with(['employees' => function ($query) {
                            $query->whereNull('end_date')->inDepartment($this->department_id);
                  -->     }])     <--
                        ->get(),
```
Tlint is wanting ],]) but when I add this my php-cs-fixer file removes the comma.  The only way to keep php-cs-fixer from removing the comma is to write is like this:
},
])

Of course this extra line looks awful.  So in the situation shown above should tlint not throw this exception for this type of multiline array in laravel?  I can work on a PR if you think the tlint exception is being thrown by mistake.

**Comment by bakerkretzmar** [comment]
@delta1186 a PR addressing this would be awesome! This has bugged me for ages, it happens with nested arrays as well.
